### PR TITLE
fix: Allow empty CSV

### LINKED ISSUE #17: Header record does not exist or is empty at offset: '0'
```
The header record does not exist or is empty at offset: `0`
Occured in /home/kaare/src/ragnarok/vendor/league/csv/src/SyntaxError.php(44)
- ...
- #5 /home/kaare/src/ragnarok/vendor/league/csv/src/Reader.php(322): League\Csv\Reader->getRecords()
- #6 /home/kaare/src/ragnarok/vendor/ragnarok/sink/src/Services/CsvToTable.php(226): League\Csv\Reader->getIterator()
- #7 /home/kaare/src/ragnarok-consat/src/Services/ConsatImporter.php(42): Ragnarok\Sink\Services\CsvToTable->exec()
- #8 /home/kaare/src/ragnarok-consat/src/Sinks/SinkConsat.php(60): Ragnarok\Consat\Services\ConsatImporter->import() 
```

### PR DESCRIPTION
**Pull request by tfk-kaare**
Imported CSVs are sometimes empty. This adds a test for it by checking if there is a header and reports a notice in the log file when empty.

### CODE REVIEW

File: src/Services/CsvToTable.php [hunk 1]
```diff
@@ -237,13 +241,6 @@ public function exec()
...
                     $this->error("Failed %d times. Halting further processing", $errCount);
                     $this->feeder->flush();
-                    $this->error(
-                        "%s(%d): %s\n%s",
-                        $except->getFile(),
-                        $except->getLine(),
-                        $except->getMessage(),
-                        $except->getTraceAsString()
-                    );
                     throw $except;
```
Comment (tfk-kaare) [inline src/Services/CsvToTable.php:244]: This is context leakage: There is no need to write this error/exception to log. It's handled by laravel

**Comment by jbtfk** [comment]
This shows up as a notice in the UI?
```php
$this->notice("Malformed or empty csv. Won't import");
```

**Comment by tfk-kaare** [comment]
No, it is from a trait that adds shorthand functions around `Log::notice(sprintf(…))` and `warning(…)`, etc. It adds stuff to the laravel log file :-)

**Review by jbtfk — APPROVED** [review]

**Review by tft-kyrre — APPROVED** [review]

**Comment by tfk-kaare** [comment]
squashed and merged locally, then pushed to main.

**Pull request closed without merging**